### PR TITLE
Chore/#181339608 setup api docummentation

### PR DESCRIPTION
**Pull request by Dev-nkundineza**
# What does this PR do?
 This is PR for setting up API documentation using swagger 
# Description of Task to be completed?

- [x] setup API documentation using swagger 

# How should this be manually tested?
clone the repository and run dev(yarn dev )     navigate    /docs/swagger-ui
## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Any background context you want to provide?

# What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2555282/stories/181412663
# Screenshots (if appropriate)

# Questions:

### CODE REVIEW

**Review by ihonore — APPROVED** [review]

**Review by gihozoinnocente — APPROVED** [review]

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-DB_URL_DEV=postgres://postgres:Password1!@127.0.0.1:5432/dev_barefoot_db
-DB_URL_TEST=postgres://postgres:Password1!@127.0.0.1:5432/test_barefoot_db
-DB_URL_PROD=postgres://postgres:Password1!@127.0.0.1:5432/prod_barefoot_db
+DB_URL_DEV=postgres://postgres:admin123@@127.0.0.1:5432/dev_barefoot_db
```
Comment (CYIMANA-Faisal) [inline .env]: Don't push the environment variables

File: swagger.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (CYIMANA-Faisal) [inline swagger.json:1]: Can we use the bearer for the authentication and authorization

Comment (CYIMANA-Faisal) [reply swagger.json:1]: Also remember to persist the token

**Review by CYIMANA-Faisal — CHANGES_REQUESTED** [review]
Good work man can you work on the reviews

File: swagger.json (hunk 2 above)
Comment (Dev-nkundineza) [reply swagger.json:1]: Thanks for the feedback, it is done.

**Pull request merged**